### DISCUSSION TITLE
@prisma/client/edge did not initialize yet. You need to either enable Accelerate or the Data Proxy.

### DISCUSSION THREAD

**Discussion by Danubius1st** [prisma/prisma#20483] [Q&A]
I have an Nextjs app deployed to vercel.
At runtime I get this error:
```
Error: @prisma/client/edge did not initialize yet. You need to either enable Accelerate or the Data Proxy.
Enable Accelerate via `prisma generate --accelerate` or the Data Proxy via `prisma generate --data-proxy.`

```
Which is not true.
/lib/prisma.ts:
`import { PrismaClient } from '@prisma/client/edge';
`
/prisma/schema.prisma:
```
datasource db {
  provider     = "mysql"
  url          = env("PrismaDB")
  directUrl    = env("DATABASE_DEV_URL")
  relationMode = "prisma"
}
```
where `PrismaDB=prisma://...`
package.json:
```
  "scripts": {
        "vercel-data-proxy-build": "npx prisma generate --data-proxy && next build",
```
What's wrong ?

I have to say that on localhost the app works well with PrismaDB.

**Comment by janpio** [comment]
User also opened an issue: https://github.com/prisma/prisma/issues/20485

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
